src/toil/batchSystems/kubernetes.py

```diff
@@ -470,11 +701,41 @@ def _create_pod_spec(
         requirements_dict = {'cpu': job_desc.cores,
                              'memory': job_desc.memory + 1024 * 1024 * 512,
                              'ephemeral-storage': job_desc.disk + 1024 * 1024 * 512}
-        # Use the requirements as the limits, for predictable behavior, and because
-        # the UCSC Kubernetes admins want it that way.
-        limits_dict = requirements_dict
-        resources = kubernetes.client.V1ResourceRequirements(limits=limits_dict,
-                                                             requests=requirements_dict)
+
+        # Also start on the placement constraints
+        placement = KubernetesBatchSystem.Placement()
+        placement.set_preemptable(job_desc.preemptable)
+
+        for accelerator in job_desc.accelerators:
+            # Add in requirements for accelerators (GPUs).
+            # See https://kubernetes.io/docs/tasks/manage-gpus/scheduling-gpus/
+
+            if accelerator['kind'] == 'gpu':
+                # We can't schedule GPUs without a brand, because the
+                # Kubernetes resources are <brand>.com/gpu. If no brand is
+                # specified, default to nvidia, which is very popular.
+                vendor = accelerator.get('brand', 'nvidia')
+                key = f'{accelerator["brand"]}.com/{accelerator["kind"]}'
```

Comment:
If `brand` is Optional, I think `accelerator["brand"]` was probably meant to be `vendor` here.